Makefile

```
@@ -1,6 +1,21 @@
-.PHONY: all rust
+.PHONY: all rust go go-deps install-protoc clean
 
-all: rust
+all: rust go
 
 rust:
 	cargo build
+
+go: install-protoc install-protobuf-go-plugins go-deps
+	./scripts/generate-go.sh
+
+go-deps:
+	go mod download
+
+install-protoc:
+	./scripts/install-protoc.sh
+
+install-protobuf-go-plugins:
+	./scripts/install-protobuf-go-plugins.sh
+
+clean:
+	rm -rf bin
```

cautious for `rm -rf / bin`

```suggestion
	rm -r bin
```

---

I use the container `namely/protoc-all:1.51_1` to compile the protobuf codes and remove the installation.